{{cookiecutter.python_name}}/pyproject.toml

```diff
@@ -76,3 +76,10 @@ build_dir = "{{cookiecutter.python_name}}/labextension"
 
 [tool.jupyter-releaser.options]
 version_cmd = "hatch version"
+
+[tool.jupyter-releaser.hooks]
+before-build-npm = [
+    "python -m pip install jupyterlab>=4.0.0a29,<5",
+    "jlpm"
```

Comment:
I think this one is missing the `jlpm build:prod` as mentioned in https://github.com/jupyterlab/extension-cookiecutter-ts/pull/251.

(was planning to close this PR and backport https://github.com/jupyterlab/extension-cookiecutter-ts/pull/251 directly)

Reply:
I saw this one then 251. I opened #252 to fix this.